### DISCUSSION TITLE
What if I want to work with IN and OZS ?

### DISCUSSION THREAD

**Discussion by mastmaster** [dvdoug/BoxPacker#257] [General]
Hello again,

So far I can understand that we pass mm(millimeters) and g(grams) as parameters and all these values must be INTEGER

What about if I want to use the American measurements system? Results will be the same right?

**Comment by shakethatweight-simon** [comment]
The algorithms don't use units, just integers, so if you supply ozs for weight, then you will see ozs in the packed object too. Same for inches, feet, yards, ,miles or any other unit you want to use. This does mean you could see a total packed weight of 1000 oz , it's not going to convert it into lbs and ozs for you.

**Comment by mastmaster** [comment]
Thank you for making it clear ;)

**Comment by mastmaster** [comment]
Ok, I have to use the pounds system.

I want to set that the empty package weight is 1.2 pounds.

Is the algorithm will get as 1 right?

What is the proper way to handle decimals? multiple * 100 when i pass and then divide / 100 on results ?

Thank you

**Reply by dvdoug** [reply]
Depends on how many decimal places you have, but yes multiplying by 10 or 100 is the way to go
